### PR TITLE
Assignment: StudentRegistry & OwnAble tests

### PR DESCRIPTION
**Pull request by RichoKD**

### CODE REVIEW

**Review by ibsule — CHANGES_REQUESTED** [review]
Can you please sync with the latest changes in the `main` so the CI can be run on your PR?

File: tests/test_student_registry.cairo [hunk 1]
```diff
@@ -1 +1,113 @@
...
+fn test_add_student() {
+    let contract_address = deploy("StudentRegistry");
+
+    let student_registry_dispatcher = IStudentRegistryDispatcher { contract_address };
+
+    let account1 = Accounts::account1();
+    let account2 = Accounts::account2();
+
+    student_registry_dispatcher.add_student('JK', account1, 20, 100, true);
+    student_registry_dispatcher.add_student('Jane', account2, 21, 101, true);
```
Comment (princeibs) [inline tests/test_student_registry.cairo:59]: After adding the student, can you add assertion statements similar to the image below that verifies the student was added successfully?
image

File: tests/test_student_registry.cairo [hunk 2]
```diff
@@ -1 +1,113 @@
...
+
+    let student_registry_dispatcher = IStudentRegistryDispatcher { contract_address };
+
+    let account1 = Accounts::account1();
+
+    student_registry_dispatcher.add_student('John', account1, 20, 100, true);
+
+    let student1 = student_registry_dispatcher.update_student('Jane', account1, 21, 101, false);
+
+    assert(student1 == true, 'Student updated failed');
```
Comment (princeibs) [inline tests/test_student_registry.cairo:97]: Can you use the updated properties of the student e.g `name` to verify the student details was updated successfully?

File: tests/test_student_registry.cairo [hunk 3]
```diff
@@ -1 +1,113 @@
...
+
+    let student_registry_dispatcher = IStudentRegistryDispatcher { contract_address };
+
+    let account1 = Accounts::account1();
+
+    student_registry_dispatcher.add_student('John', account1, 20, 100, true);
+
+    let student1 = student_registry_dispatcher.delete_student(account1);
+
+    assert(student1 == true, 'Student deleted failed');
```
Comment (princeibs) [inline tests/test_student_registry.cairo:112]: Please add assertions to verify the student was deleted successfully

**Review by ibsule — CHANGES_REQUESTED** [review]
Please let me know if you need more clarity on the reviews

**Pull request closed without merging**